src/collections/_documentation/clients/go/context.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Context
+sidebar_order: 3
+---
+
+All of the `Capture*` functions accept an additional argument for passing a `map` of tags as the second argument and context data as remaining ones.
+Tags in Sentry help to categories and give you more information about the errors that happened, where other context data is more "environment" specific.
+(NOTE: direct low-level `Capture` call accepts second argument only - tags, without additional interfaces)
+
+For example:
+
+```go
+raven.CaptureError(err, map[string]string{"browser": "Firefox"}, &raven.Http{
+  Method: "GET",
+  URL: "https://example.com/raven-go"
+})
+```
+
+This data can be either passed directly to `Capture*` calls or configured globally using functions listed below, if it should be applied to all future events.
+
+{:.config-key}
+### `SetHttpContext`
+
+Provides information about an HTTP call that was processed when error happened.
```

Comment:
Fixed passive voice to active voice. Here's the edit, "Provides information about an HTTP call that the Sentry SDK processes when the error happens."